### PR TITLE
major correction of mistakes in the French translation

### PR DESCRIPTION
**Pull request by etlel**
I corrected a lot of mistakes and inconsistencies in the French translation files. It may not be perfect at this stage but it is far more better than previously. I am pleased to see this contribution integrated as soon as possible.

Regards,

### CODE REVIEW

File: lam/locale/fr_FR/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -830,7 +828,7 @@ msgstr "Ajouter l'extension FreeRadius"
 
 #: ../lib/modules/ipHost.inc:177
 msgid "Add IP address extension"
-msgstr "Ajouter l'extension adresse IP"
+msgstr "Ajouter l'extension Adresse IP"
```
Comment (gruberroland) [inline lam/locale/fr_FR/LC_MESSAGES/messages.po]: Can you recheck this one? E.g. here it is as it was before: https://fr.wikipedia.org/wiki/Adresse_IP

File: lam/locale/fr_FR/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -810,11 +808,11 @@ msgstr "Ajouter compte Asterisk"
 
 #: ../lib/modules/asteriskVoicemail.inc:269
 msgid "Add Asterisk voicemail extension"
-msgstr "Ajouter les extensions de boîte vocale Asterisk"
+msgstr "Ajouter l'extensions de boîte vocale Asterisk"
```
Comment (gruberroland) [inline lam/locale/fr_FR/LC_MESSAGES/messages.po]: Should be "l'extension" (singular)?

File: lam/locale/fr_FR/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -4051,7 +4044,7 @@ msgid ""
 "Displayed account numbers start at \"0\". Add 2 to get the row in your "
 "spreadsheet."
 msgstr ""
-"Numéros de comptes affichés commencent à \"0\".Ajoute 2 pour obtenir la "
+"Numéros de comptes affichés commencent à \"0\". Ajouter 2 pour obtenir la "
```
Comment (gruberroland) [inline lam/locale/fr_FR/LC_MESSAGES/messages.po]: Please recheck if this should be Ajoute/Ajoutez.

File: lam/locale/fr_FR/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -4661,11 +4648,11 @@ msgstr "Entrée"
 #: ../templates/3rdParty/pla/htdocs/rdelete.php:43
 #, php-format
 msgid "Entry %s and sub-tree deleted successfully."
-msgstr "Entrée %s et sous-arbre effacée correctement."
+msgstr "Entrée %s et sous-arbre effacés correctement."
 
 #: ../templates/3rdParty/pla/htdocs/create.php:90
 msgid "Entry created"
-msgstr "Entrée créé"
+msgstr "Entrée créée"
```
Comment (gruberroland) [inline lam/locale/fr_FR/LC_MESSAGES/messages.po:4564]: Is the third e correct?

File: lam/locale/fr_FR/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -6216,9 +6198,9 @@ msgid ""
 "it will be added. If host name is already used host name will be expanded "
 "with a number. The next free number will be used."
 msgstr ""
-"Nom de machine qui doit être créé. Les caractères autorisés sont: a-z,A-"
+"Nom de la machine qui doit être créée. Les caractères autorisés sont : a-z,A-"
```
Comment (gruberroland) [inline lam/locale/fr_FR/LC_MESSAGES/messages.po:6084]: Please recheck the third e in créée.

File: lam/locale/fr_FR/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -7167,31 +7148,31 @@ msgstr "mot de passe utilisateur pour LAM"
...
 
 #: ../lib/modules/sambaGroupMapping.inc:578
 msgid "LAM was unable to find a Samba 3 domain with this name!"
-msgstr "LAM n'a pu trouver un Domaine SAMBA3 avec ce nom!"
+msgstr "LAM n'a pu trouver de domaine Samba 3 portant ce nom!"
 
 #: ../lib/modules/sambaSamAccount.inc:118
 msgid "LAM was unable to find a domain with this name!"
-msgstr "LAM n'a pas trouvé de domaine avec ce nom!"
+msgstr "LAM n'a pu trouver de domaine portant ce nom!"
```
Comment (gruberroland) [inline lam/locale/fr_FR/LC_MESSAGES/messages.po:7026]: Please recheck "pu trouver"

File: lam/locale/fr_FR/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -7385,31 +7367,31 @@ msgstr "Import LDIF"
...
 
 #: ../templates/tests/lamdaemonTest.php:186
 msgid "Lamdaemon server and path"
-msgstr "chemin de lamdaemon et serveur"
+msgstr "Serveur et chemin Lamdaemon"
 
 #: ../templates/config/confmain.php:397
 msgid "Lamdaemon settings"
-msgstr "paramètres de lamdaemon"
+msgstr "Paramètres de lamdaemon"
```
Comment (gruberroland) [inline lam/locale/fr_FR/LC_MESSAGES/messages.po]: **L**amdaemon?

File: lam/locale/fr_FR/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -7422,11 +7404,11 @@ msgstr "tester lamdaemon"
 
 #: ../templates/tests/lamdaemonTest.php:290
 msgid "Lamdaemon test finished."
-msgstr "test de lamdaemon terminé."
+msgstr "Test de Lamdaemon terminé."
 
 #: ../templates/tests/lamdaemonTest.php:275
 msgid "Lamdaemon version"
-msgstr "version de lamdaemon"
+msgstr "version de Lamdaemon"
```
Comment (gruberroland) [inline lam/locale/fr_FR/LC_MESSAGES/messages.po]: Should be "**V**ersion"?

File: lam/locale/fr_FR/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -2928,15 +2925,14 @@ msgstr "Impossible de sauvegarder la structure PDF, accès refusé."
 #: ../lib/modules/courierMailAlias.inc:56
 #: ../lib/modules/courierMailAccount.inc:72
 msgid "Courier"
-msgstr "Courier"
+msgstr "Courrier"
```
Comment (gruberroland) [inline lam/locale/fr_FR/LC_MESSAGES/messages.po]: Should stay "Courier" as it is the name of a software.

**Review by gruberroland — CHANGES_REQUESTED** [review]
Merci beaucoup! I found some potential issues using my little French knowledge from school.
Please recheck them if any changes are required.

Anyway, looks like you solved a lot of issues. Thanks a lot for your work. :)

File: lam/locale/fr_FR/LC_MESSAGES/messages.po (hunk 1 above)
Comment (etlel) [reply lam/locale/fr_FR/LC_MESSAGES/messages.po]: As it is the extension "name". But another version could be "Ajouter l'extension d'adresse IP" and that would indeed be consistent with other related messages.

File: lam/locale/fr_FR/LC_MESSAGES/messages.po (hunk 2 above)
Comment (etlel) [reply lam/locale/fr_FR/LC_MESSAGES/messages.po]: I changed all these mentions of specific extensions to singular. Forgot half of this one.

File: lam/locale/fr_FR/LC_MESSAGES/messages.po (hunk 4 above)
Comment (etlel) [reply lam/locale/fr_FR/LC_MESSAGES/messages.po:4564]: Yes it is right :) https://fr.wiktionary.org/wiki/cr%C3%A9%C3%A9e

File: lam/locale/fr_FR/LC_MESSAGES/messages.po (hunk 6 above)
Comment (etlel) [reply lam/locale/fr_FR/LC_MESSAGES/messages.po:7026]: "n'a pas trouvé" = didn't find; "n'a (pas) pu trouver"= "couldn't find". I changed some messages for consistency reasons.

**Comment by etlel** [comment]
Before merging, I'd like to review the translations related to "block". I have been confused by the potential multiple meanings that seems to being used. For instance, I believe that one is [block](https://en.wikipedia.org/wiki/Block_(data_storage)) (in "Block soft quota"; _bloc_ in French). But also [blocking](https://en.wikipedia.org/wiki/Wikipedia:Blocking_policy) (in "Grace block period"; _blocage_ in French). So I need more context around less precise strings like "Hard inode" or "Hard inode limit" to be sure that _hard_ is about the limit or the inode... and so on for "Hard block", "Hard block limit"...

**Comment by gruberroland** [comment]
In this context, block is a block in the file system. So not related to blocking. Also in grace block period it is about a file system block.
In "Hard inode (limit)" the hard is about the limit. There is a soft and a hard limit. Same for hard block (limit).
Please let me know if you need more details.

**Comment by gruberroland** [comment]
@etlel Can you update the pull request or do you need any additional information?

**Comment by gruberroland** [comment]
Thanks a lot for your update! :)

**Pull request merged**